spyder/plugins/toolbar/container.py

```diff
@@ -10,7 +10,7 @@
 
 # Standard library imports
 from collections import OrderedDict
-from spyder.utils.qthelpers import SpyderAction
+import logging
```

Comment:
Shouldn't `import logging` go first/before `from collections import OrderedDict` too?

Reply:
Our custom is to organize `from` imports and regular imports alphabetically and mixed together, that's why I made this change like this.